pkg/database/query.go

```diff
@@ -0,0 +1,154 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package database
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"reflect"
+	"strings"
+)
+
+type paramQuery struct {
+	query         string
+	arguments     []interface{}
```

Comment:
You can use `any`, which sometimes is "plain english" than `interface{}`:


```suggestion
	arguments     []any
```

However it is not a big deal, at the end of the day both are used widely.

Reply:
I changed my mind several times of using the alias over the native name or the other way around but at the end I have both and I see that are using the alias in other parts of the code so I'll change to the alias `any` for consistency.